common/src/main/java/com/microsoft/identity/common/internal/cache/BrokerOAuth2TokenCache.java

```diff
@@ -302,10 +302,15 @@ public ICacheRecord save(@NonNull final GenericOAuth2Strategy oAuth2Strategy,
         if (isFoci) {
             targetCache = mFociCache;
         } else {
+
+            // Suppressing unchecked warning as the generic type was not provided for oAuth2Strategy
+            @SuppressWarnings(WarningType.unchecked_warning)
+            final String environment = oAuth2Strategy.getIssuerCacheIdentifier(request);
```

Comment:
Is this authority or environment?  Just not sure what this method returns hence the question.  No change to the code though... so not worried about this.

Reply:
Hi Shane, this is because of the request parameter of type GenericAuthorizationRequest and not due to the environment. I'll mention this in the comment in the next PR.

Thanks